SmartCmdArgs/SmartCmdArgs.Shared/ViewModel/SettingsViewModel.cs

```diff
@@ -38,9 +57,19 @@ public class SettingsViewModel : PropertyChangedBase
             set => SetAndNotify(value, ref _macroEvaluationEnabled);
         }
 
-        public SettingsViewModel() { }
+        public RelayCommand OpenOptionsCommand { get; }
```

Comment:
I have a feeling this is from a bad merge. I didn't intend to modify this constructor at all.